scss/civicrm/common/_replace-plain.scss

```diff
@@ -0,0 +1,8 @@
+.replace-plain {
```

Comment:
what's this for? is this a custom class or a civi class?

Reply:
This is for inputs 
![replace-plain](https://user-images.githubusercontent.com/3272950/35628012-4f66249c-06a3-11e8-9e31-91284569f9f9.png)
Not sure if it is custom or civi class...
When you click, this input expands to:
![replace-expanded](https://user-images.githubusercontent.com/3272950/35628092-828ee2e6-06a3-11e8-9beb-8b02d8208b86.png)
And when focus out it change back to simple input